packages/package_info_plus/package_info_plus/lib/src/package_info_plus_web.dart

```diff
@@ -4,7 +4,7 @@ import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 import 'package:http/http.dart';
 import 'package:package_info_plus_platform_interface/package_info_data.dart';
 import 'package:package_info_plus_platform_interface/package_info_platform_interface.dart';
-import 'package:web/helpers.dart' as web;
```

Comment:
I think you need to leave helpers.dart, because we want to support web 0.3

Reply:
oh, I will revert it back if the PR open again

Reply:
reviewer made an alternative PR already